tests/bootstrap/environment.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+// Bootstrap _ss_environment.php
+
+if (!defined('SS_ENVIRONMENT_TYPE')) {
+	define('SS_ENVIRONMENT_TYPE', 'dev');
+}
+
+if (!defined('SS_DATABASE_CLASS') && !defined('SS_DATABASE_USERNAME')) {
```

Comment:
Future idea: allow these `define()`s to be set by environment variables (probably as part of ConfigureFromEnv) and then put the relevant defines explicitly into .travis.yml